graph/resolver/discussion.resolvers.go

```diff
@@ -132,6 +132,61 @@ func (r *discussionResolver) UpcomingContent(ctx context.Context, obj *model.Dis
 	return r.DAOManager.GetUpcomingImportedContentByDiscussionID(ctx, obj.ID)
 }
 
+func (r *discussionResolver) FlairTemplates(ctx context.Context, obj *model.Discussion) ([]*model.FlairTemplate, error) {
+	return r.DAOManager.GetDiscussionFlairTemplateAccessByDiscussionID(ctx, obj.ID)
+}
+
+func (r *discussionResolver) AccessRequests(ctx context.Context, obj *model.Discussion) ([]*model.DiscussionAccessRequest, error) {
+	authedUser := auth.GetAuthedUser(ctx)
+	if authedUser == nil {
+		return nil, fmt.Errorf("Need auth")
+	}
+
+	// Only allow the mod to view access requests
+	modCheck, err := r.DAOManager.CheckIfModeratorForDiscussion(ctx, authedUser.UserID, obj.ID)
+	if err != nil || !modCheck {
+		return nil, fmt.Errorf("unauthorized")
+	}
+
+	return r.DAOManager.GetDiscussionAccessRequestsByDiscussionID(ctx, obj.ID)
+}
+
+func (r *discussionAccessRequestResolver) User(ctx context.Context, obj *model.DiscussionAccessRequest) (*model.User, error) {
```

Comment:
I _think_ we want to gate this to only the user who is being invited

Reply:
good call. it should be gated to the requesting user and mod of the discussion

Reply:
updated